src/Console/Deployer.php

```diff
@@ -59,7 +59,7 @@ function ($value, $key) {
             array_keys($parameters)
         ));
 
-        $process = new Process('serverless invoke local ' . $p, '.bref/output');
+        $process = new Process(['serverless invoke local ' . $p, '.bref/output']);
```

Comment:
I guess this should be

```suggestion
        $process = new Process(['serverless invoke local ' . $p], '.bref/output');
```